apollo-federation/src/query_plan/operation.rs

```diff
@@ -155,24 +145,24 @@ fn same_directives(left: &DirectiveList, right: &DirectiveList) -> bool {
 /// - Stores the fragments used by this operation (the executable document the operation was taken
 ///   from may contain other fragments that are not used by this operation).
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct NormalizedOperation {
+pub struct Operation {
     pub(crate) schema: ValidFederationSchema,
     pub(crate) root_kind: SchemaRootDefinitionKind,
     pub(crate) name: Option<Name>,
-    pub(crate) variables: Arc<Vec<Node<VariableDefinition>>>,
-    pub(crate) directives: Arc<DirectiveList>,
-    pub(crate) selection_set: NormalizedSelectionSet,
+    pub(crate) variables: Arc<Vec<Node<executable::VariableDefinition>>>,
+    pub(crate) directives: Arc<executable::DirectiveList>,
+    pub(crate) selection_set: SelectionSet,
     pub(crate) named_fragments: NamedFragments,
 }
 
 pub(crate) struct NormalizedDefer {
```

Comment:
This `Normalized` is an unrelated kind of `Normalized` from the selection and operation types, so I kept it.